### PR TITLE
Update DEVELOPERS.md

### PR DESCRIPTION
**Pull request by gargarchit**
Signed-off-by: Archit garg4.ag@gmail.com

I found some Improper word choice as well as Improper formatting for that I have made changes and corrected that so, now It will be easier for everyone to read.
@irmerk Please checkout this

### CODE REVIEW

**Comment by irmerk** [comment]
Linking https://github.com/accordproject/techdocs/issues/226

Link this in any further PRs

**Comment by gargarchit** [comment]
@irmerk Please Review this

**Comment by gargarchit** [comment]
Now this will be fine also this time I take care of previous Reviews

File: DEVELOPERS.md [hunk 1]
```diff
@@ -129,10 +129,10 @@ Formatting for the body is displayed in this example:
 
 ### Changes
 - Change one
-  - Subchange one
-  - Subchange two
+  - Sub change one
+  - Sub change two
```
Comment (irmerk) [inline DEVELOPERS.md]: This can be removed.

File: DEVELOPERS.md [hunk 2]
```diff
@@ -69,7 +69,7 @@ We have very precise rules over how our git commit messages can be formatted.  T
 readable messages** that are easy to follow when looking through the **project history** and **git logs**.  
 But also, we use the git commit messages to **generate the Cicero Template Library change log**.
 
-The commit message formatting can be added using a version of typical git workflow.
+The commit message formatting can be added using a version of the typical git workflow.
```
Comment (irmerk) [inline DEVELOPERS.md]: This can be removed.

File: DEVELOPERS.md [hunk 3]
```diff
@@ -30,12 +30,12 @@ To contribute code to Cicero Template Library, you must have a GitHub account so
...
 
 To create a Github account, follow the instructions [here][github-signup].
-Afterwards, go ahead and [fork][github-forking] the
+Afterward, go ahead and [fork][github-forking] the
 [main Cicero Template Library repository][github].
 
 ### Keeping In Sync
 
-It is good practice to always keep your `origin/master` in sync with `upstream/master`. You don’t have to, but it makes your life easier. Do your work in branches of your fork, and periodically sync up your `master` with the `master` of `upstream` as follows. You should definitely do this before creating a pull request.
+It is good practice to always keep your `origin/master` in sync with `upstream/master`. You don’t have to, but it makes your life easier. Do your work in branches of your fork, and periodically sync up your `master` with the `master` of `upstream` as follows. You should do this before creating a pull request.
```
Comment (irmerk) [inline DEVELOPERS.md]: This can be removed.

**Comment by jeromesimeon** [comment]
No activity on this. I'm closing the PR.

**Pull request closed without merging**